controlplane/src/core/repositories/FederatedGraphRepository.ts

```diff
@@ -503,7 +518,13 @@ export class FederatedGraphRepository {
         .select({ id: schemaVersion.id })
         .from(schemaVersion)
         .innerJoin(schemaVersionChangeAction, eq(schemaVersionChangeAction.schemaVersionId, schemaVersion.id))
-        .where(eq(schemaVersion.targetId, targetId))
+        .where(
+          and(
+            eq(schemaVersion.targetId, targetId),
+            gt(schemaVersion.createdAt, new Date(start)),
```

Comment:
Are you sure it will be compared in UTC? You validate it in the query logs.

Reply:
Yes